### PR TITLE
Remove tabs

### PR DESCRIPTION
**Pull request by rosemarymarano**
As per [VZ-9225](https://jira.oraclecorp.com/jira/browse/VZ-9225) Remove tabs in docs

### CODE REVIEW

File: content/en/docs/samples/sock-shop.md [hunk 1]
```diff
@@ -46,11 +46,9 @@ It uses OAM resources to define the application deployment:
 </div>
 {{< /clipboard >}}
 
-1. To deploy the application, apply the Sock Shop OAM resources.  Choose to deploy either the `helidon`, `micronaut`, or `spring` variant.
+1. To deploy the application, apply the Sock Shop OAM resources.  Choose to deploy either the `helidon`, `micronaut`, or `spring` variant:
 
-   {{< tabs tabTotal="3" >}}
-   {{< tab tabName="Helidon" >}}
-   <br>
+`helidon`
```
Comment (MelwynPaul) [inline content/en/docs/samples/sock-shop.md:51]: The formatting is not correct here. 
The tab titles were in code-font. We have just removed the tab code. 

The tab headings and indentation should be updated. 

Screenshot of the output: 

image

File: content/en/docs/samples/sock-shop.md [hunk 2]
```diff
@@ -268,10 +260,8 @@ You can access them according to the directions [here]({{< relref "/docs/access/
 
 1. To undeploy the application, delete the Sock Shop OAM resources.  Choose to undeploy either the `helidon`, `micronaut`, or `spring` variant.
 
+`helidon`
```
Comment (MelwynPaul) [inline content/en/docs/samples/sock-shop.md:263]: Same as the previous comment on this topic. 
Formatting should be corrected. Please check the output.

File: content/en/docs/setup/private-registry/private-registry-full-distribution.md [hunk 3]
```diff
@@ -28,12 +28,12 @@ You must have the following software installed:
...
-Set up a private registry using the following instructions, depending on your distribution.
-{{< tabs tabTotal="2" >}}
-{{< tab tabName="LiteDistribution" >}}
-<br>
+Set up a private registry using the following instructions, depending on your distribution:
 
-## Load the images
+- [Lite Distribution](#lite-distribution)
+- [Full Distribution](#full-distribution)
+
```
Comment (MelwynPaul) [inline content/en/docs/setup/private-registry/private-registry-full-distribution.md:35]: 1 - _Existing issue, after tab module/something changed_: Lite Distribution: 
        - Formatting after Step 1c needs correction
        - Step 2 says Step 1
        
2 - For both the Lite and Full Distribution sections, tabs were used to differentiate Steps 1 and 2 
      Step 3 and its sub steps  - "Load the product images into your private registry" - which was outside the tabs was common
      After removing the tabs, Lite Distribution sections stops with Step 2

File: content/en/docs/upgrade/upgrade.md [hunk 4]
```diff
@@ -35,11 +35,10 @@ See the following respective sections.
...
 {{< /alert >}}
 
-{{< tabs tabTotal="2" >}}
-{{< tab tabName="vz" >}}
-<br>
+- [Upgrade Verrazzano using the CLI](#upgrade-verrazzano-using-the-cli)
+- [Upgrade using `kubectl`](#upgrade-using-kubectl)
 
-## Upgrade Verrazzano
+## Upgrade Verrazzano using the CLI
```
Comment (MelwynPaul) [inline content/en/docs/upgrade/upgrade.md:42]: Fix formatting under Step 1

File: content/en/docs/upgrade/upgrade.md [hunk 5]
```diff
@@ -71,9 +70,7 @@ In one simple step, you can upgrade to a specified version of Verrazzano using t
    Upgrade logs will be streamed to the command window until the upgrade has completed
    or until the default timeout (30m) has been reached.
 
-{{< /tab >}}
-{{< tab tabName="kubectl" >}}
-<br>
+## Upgrade using `kubectl`
```
Comment (MelwynPaul) [inline content/en/docs/upgrade/upgrade.md:74]: Fix formatting under the "Upgrade Verrazzano" sub section.

**Review by MelwynPaul — COMMENTED** [review]
Added a few comments.

File: content/en/docs/samples/sock-shop.md (hunk 1 above)
Comment (rosemarymarano) [reply content/en/docs/samples/sock-shop.md:51]: Yep, I know about this, just couldn't get it right. I'll keep trying today.

File: content/en/docs/samples/sock-shop.md (hunk 2 above)
Comment (rosemarymarano) [reply content/en/docs/samples/sock-shop.md:263]: Saw it but wasn't able to fix it yet. Will try again.

File: content/en/docs/samples/sock-shop.md (hunk 1 above)
Comment (rosemarymarano) [reply content/en/docs/samples/sock-shop.md]: Awesome. Thank you @MelwynPaul

File: content/en/docs/samples/sock-shop.md (hunk 2 above)
Comment (rosemarymarano) [reply content/en/docs/samples/sock-shop.md]: You did it, thanks!!!

**Review by MelwynPaul — APPROVED** [review]
All looks good. 
Thank you.
Approved.

**Pull request merged**